src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp

```diff
@@ -391,6 +391,47 @@ std::set<std::vector<element::Type>> jit_exp_emitter::get_supported_precisions(c
     return {{element::f32}};
 }
 
+/// MOD ///
+jit_mod_emitter::jit_mod_emitter(dnnl::impl::cpu::aarch64::jit_generator *host,
+                                 dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                 const std::shared_ptr<ov::Node>& node)
+    : jit_emitter(host, host_isa, node, get_arithmetic_binary_exec_precision(node)) {
+}
+
+jit_mod_emitter::jit_mod_emitter(dnnl::impl::cpu::aarch64::jit_generator *host,
+                                         dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                         const ov::element::Type exec_prc): jit_emitter(host, host_isa, exec_prc) {
+}
+
+size_t jit_mod_emitter::get_inputs_count() const { return 2; }
+
+void jit_mod_emitter::emit_impl(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const {
+    if (host_isa_ == dnnl::impl::cpu::aarch64::asimd) {
+        emit_isa<dnnl::impl::cpu::aarch64::asimd>(in_vec_idxs, out_vec_idxs);
+    } else {
+        OV_CPU_JIT_EMITTER_THROW("Can't create jit eltwise kernel");
+    }
+}
+
+template <dnnl::impl::cpu::aarch64::cpu_isa_t isa>
+void jit_mod_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const {
+    OV_CPU_JIT_EMITTER_ASSERT(exec_prc_ == ov::element::f32, "unsupported precision: " + exec_prc_.to_string());
+
+    using TReg = typename dnnl::impl::cpu::aarch64::cpu_isa_traits<isa>::TReg;
+
+    TReg divend = TReg(in_vec_idxs[0]);
+    TReg divisor = TReg(in_vec_idxs[1]);
+    TReg r = TReg(out_vec_idxs[0]);
+
+    h->uni_fdiv(r.s, divend.s, divisor.s);
+    h->uni_fmul(r.s, r.s, divisor.s);
+    h->uni_fsub(r.s, divend.s, r.s);
```

Comment:
Regarding your question in https://github.com/openvinotoolkit/openvino/issues/24061#issuecomment-2118860639:
This emitter works with FP32 values. It means that `uni_fmul` instruction is inverse for the previous instruction `uni_fdiv`.
For example,
```
divend = 10.f
divisor = 3.f
h->uni_fdiv(r.s, divend.s, divisor.s); // r.s = 10.f / 3.f = 3.(33)f
h->uni_fmul(r.s, r.s, divisor.s); // r.s = 3.(33)f * 3.f ~= 10.f
h->uni_fsub(r.s, divend.s, r.s); // r.s = 10.f - 10.f = 0
```
So you need to truncate fractional part of division result before `fmul` call.

Reply:
@a-sidorova  Thank you very much for your detailed suggestions. I have incorporated your suggestions and passed the tests. Could you please review the PR?